examples/basics/custom_embeddings_sdk.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
It would have been nice if the example helped motivate the need for custom embeddings.

Reply:
Is this not sufficient (see top-most paragraph)?

https://github.com/Labelbox/labelbox-python/blob/mrobers1982/PLT_582/examples/basics/custom_embeddings_sdk.ipynb

Reply:
There is also customer-facing documentation on our website - https://docs.labelbox.com/docs/embeddings.

It ultimately links here, not sure we need to duplicate everywhere.